MediaService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace HNDunit20
+{
+    class MediaService : ServiceInterface
+    {
+        public double SongPrice
+        {
+            get { return 2.5; }
+        }
+
+        public IList<string> Songs
+        {
+            get
+            {
+                return new List<string>
+                {
+                    "Song 1",
+                    "Song 2",
+                    "Song 3"
+                };

```

Comment:
I think we're fine to just initialize a list of songs here so they're available at runtime. Unless there is a requirement to update the available songs while the  app is running I don;t think we need a setter